stubs/SQLAlchemy/sqlalchemy/future/engine.pyi

```diff
@@ -1,11 +1,25 @@
-from typing import Any
+from typing import Any, overload
+from typing_extensions import Literal
 
 from ..engine import Connection as _LegacyConnection, Engine as _LegacyEngine
 from ..engine.base import OptionEngineMixin
+from ..engine.mock import MockConnection
+from ..engine.url import URL
 
 NO_OPTIONS: Any
 
-def create_engine(*arg, **kw): ...
+@overload
+def create_engine(url: URL | str, *, strategy: Literal["mock"], **kwargs) -> MockConnection: ...  # type: ignore[misc]
+@overload
+def create_engine(
+    url: URL | str,
+    *,
+    empty_in_strategy: object | None = ...,
```

Comment:
This argument is deprecated. I don't think we should include it in the stubs, so that it won't show up in autocompletions. It can still be used with `**kwargs`.